### DISCUSSION TITLE
建议使用Girhub的Release API来替代私有的更新服务！

### DISCUSSION THREAD

**Discussion by cooaer** [lollipopkit/flutter_server_box#112] [Ideas]
API地址：https://docs.github.com/en/rest/releases?apiVersion=2022-11-28
出于一些安全和服务稳定性的考虑，Github的API可能是更好的选择！

**Comment by lollipopkit** [comment]
大部分使用者在国内，GitHub 访问不畅通。
后继会校验 hash。
